docs/docs/api/06-Getters/getall.md

```diff
@@ -4,5 +4,45 @@ custom_edit_url: null
 ---
 # getAll
 
-Returns all detected elements
+Returns an array with all detected elements.
+A detected element has the following properties:
+- `name` of the element
+- `text` content of element
+- `colors` of element
+- `bndbox`: location of element described with coordinates of a bounding box
+
+**Examples:**
```

Comment:
Nice that there are code examples. They really help to understand how it works! 👍